test/data.jl

```diff
@@ -189,6 +189,15 @@ module TestData
     @test d1us2[:d] == d1[:d]
     @test d1us2[:3] == d1[:d]
 
+    #
+
+    df1 = melt(DataFrame(rand(10,10)))
+    df1[:id] = 1:100
+    @test size(unstack(df1, :variable, :value)) == (100, 11)
+    # test empty keycol
```

Comment:
With the line break afterward, it's not clear what line this comment is attached to.

Reply:
I had it fixed - by error I did not push the commit. Should be OK now